plugins/civmodcore-paper/src/main/java/vg/civcraft/mc/civmodcore/CivModCorePlugin.java

```diff
@@ -88,6 +89,7 @@ public void onEnable() {
 		registerListener(DialogManager.INSTANCE);
 		registerListener(new ScoreBoardListener());
 		registerListener(new WorldTracker());
+		registerListener(PlayerNames.init());
```

Comment:
Could just `new PlayerNames()` here instead of a static method and an anonymous class